src/components/PageDefinitions.js

```diff
@@ -283,21 +308,21 @@ class PageDefinitions extends AbstractPageDefinitions {
     const { dispatch, token, definitions } = this.props
     const component = typeof value === 'string' ? EntitySpec.fromPath(value) : value
     const path = component.toPath()
+
     !definitions.entries[path] && dispatch(getDefinitionsAction(token, [path]))
     dispatch(uiBrowseUpdateList({ add: component }))
   }
 
   dropZone(child) {
     return (
-      <Dropzone
-        accept="application/json"
-        disableClick
+      <div
```

Comment:
Instead of build in place the functionality, can we make a reusable component, that address file and urls handling?